### ISSUE TITLE
Broken XL Page

### ISSUE THREAD

**Issue by RassanGrant** [Norstone/norstone.github.io#308]
@vleong2332 - We had an urgent issue on the site today. I believe the adjustment you made to the XL page yesterday ended up breaking the layout altogether. I created a new branch to fix the issue, but ran into some errors I did not understand, so I abandoned them and edited Develop directly (a no-no). 

Anyway, when you get this, can you please:

1. Correct my work (the fix should have been added to Develop by way of another branch)
2. Review the work you did previously to figure out what happened and how to correct it 

<img width="1575" alt="2018-11-02 01 23 30 pm" src="https://user-images.githubusercontent.com/18408413/47931094-33cee500-dea4-11e8-89e1-da5ee757b5aa.png">

**Comment by vleong2332** [comment]
@RassanGrant 
Here's what I believe happened.
- I made an update from Forestry that generates the `date` and `layout`.
- I confirmed that the faulty `date` value broke the wercker build.
- I put a valid value for `date` via Forestry and the wercker build passed.
- The faulty `layout` threw a warning (I found this out when building locally today) but a warning doesn't break a wercker build so I missed this.
- I checked the website for that page and it seemed fine (That's what I remember but apparently not the case).

Was there cloudflare caching that prevented me from seeing the broken page? When did you notice it?

I'm trying to think what I could've done differently to "preview" updates from Forestry. The "preview" functionality seems to not work properly (it doesn't show the css/style). The only thing I could think of is to "simulate" the change locally on my machine, which I'll do from now on when working with Forestry.
